index.js

```diff
@@ -85,8 +85,12 @@ var E, InvalidAstError = E = (function() {
 }());
 
 
-// errorsP :: {labels :: [Label], inFunc :: Boolean, inIter :: Boolean, inSwitch :: Boolean} -> Node -> [InvalidAstError]
+// errorsP :: {labels :: [Label], inFunc :: Boolean, inIter :: Boolean, inSwitch :: Boolean, ES6 :: Boolean } -> Node -> [InvalidAstError]
```

Comment:
I was changing my proposal till you agreed here: https://github.com/estools/esvalid/issues/7
No problem changing it back to... just let me know your choice.